org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/console/JavaStackTraceHyperlink.java

```diff
@@ -201,14 +236,326 @@ public IStatus runInUIThread(IProgressMonitor monitor) {
 		};
 		job.setSystem(true);
 		job.schedule();
+
+	}
+
+	/**
+	 * Process received classes to extract exact class by mining methods
+	 *
+	 * @param matches
+	 *            Initial match results
+	 * @param typeName
+	 *            the fully qualified type name
+	 * @param lineNumber
```

Comment:
please fix javadoc
@param name not found

lineNumber, status are wrong

Reply:
Will fix it